src/Symfony/Component/Console/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Exception;
+
+/**
+ * ExceptionInterface.
+ *
+ * @author Jérôme Tamarelle <jerome@tamarelle.net>
+ *
+ * @api
+ */
+interface ExceptionInterface
```

Comment:
Do we really need the interface? And do we need to tag it with as an `@api` interface given that the exception class are not tagged this way?